plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -259,11 +263,14 @@ function auto_sizes_filter_uses_context( array $uses_context, WP_Block_Type $blo
 	$consumer_blocks = array(
 		'core/cover',
 		'core/image',
+		'core/group',
+		'core/columns',
+		'core/column',
 	);
 
 	if ( in_array( $block_type->name, $consumer_blocks, true ) ) {
 		// Use array_values to reset the array keys after merging.
-		return array_values( array_unique( array_merge( $uses_context, array( 'max_alignment' ) ) ) );
+		return array_values( array_unique( array_merge( $uses_context, array( 'max_alignment', 'column_count', 'column_width' ) ) ) );
```

Comment:
Ideally this allowlist should be more granular than allowing every new context for all of these block types: For example, I think the `column_count` is only relevant between the parent `core/columns` block and its `core/column` child block. So I think only `core/column` needs to be allowed to _use_ the `column_count` context, and only the `core/columns` block will need to provide it.

Reply:
Nice cache. Updated in https://github.com/WordPress/performance/pull/1795/commits/09d49904cde646451477511b357861638663272a